lib/cc/engine/analyzers/sexp.rb

```diff
@@ -0,0 +1,3 @@
+class Sexp
+  attr_accessor :end_line
```

Comment:
This is required to get the correct end of JavaScript expressions.

Without it, the following:

```
if (foo) {
  bar();
}
```

Would show up as:

```
if (foo) {
  bar();
```

In the application.